packages/fluent-editor/src/modules/ai/index.ts

```
@@ -218,68 +715,130 @@
+
+    if (this.resultPopupEl) {
+      this.resultPopupEl.style.display = 'none'
+    }
+
+    if (this.inputEl && this.inputEl.value.trim() !== '') {
+      this.inputEl.value = '' // 清空输入框
+    }
+    this.hideSelectionBubble()
+  }
+
+  set charCount(value: number) {
```

ESLint

[error] Getter is not present for class setter 'charCount'.

(accessor-pairs)